.travis_update_ghpages.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Only publish from the main repository's master branch
+REPO_NAME="HydraCG/Specifications"
+if [ "$TRAVIS_REPO_SLUG" != "$REPO_NAME" ] || [ "$TRAVIS_BRANCH" != "master" ] || [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit; fi
+echo -e "Generating gh-pages...\n"
+
+# Checkout the gh-pages branch
+REPO_PATH=$PWD
+pushd $HOME
+git clone --quiet --branch=gh-pages https://${GH_TOKEN}@github.com/$REPO_NAME gh-pages 2>&1 > /dev/null
```

Comment:
What I mean about "hacky" is that this is a weird use of Git. But I agree that it might be a good idea to have compiled code away from `master`.